docs/FAQ/ServernGamemodes.md

```diff
@@ -0,0 +1,196 @@
+
+PvE is a dimension where you kill mobs and bosses, either for fun or to get DP and XP, it has a few warps in spawn, but if you're do lazy to walk, just do `/warp list` then do `/warp <warp name>`
+
+> Q: Why won't the bosses Spawn?
+>
+> A: You don't post the commands on the sign to chat, you just break the orange-colored sign.
+>
+> A2: There's a cooldown active, cooldowns are listed on some signs as seconds, globally signs that 60 seconds cooldown.
+
+> Q: There is a hacker named Rofle-Bot.
+>
+> A: No, that's not a hacker that's a bot to prevent Moon Lord from going elsewhere in PvE.
+
+### PvP
+TShock Server, `/pvp` to join.
+
+Our pvp is like no other, many weapons have had their damage altered, velocity altered and their damage capped. Certain equipment and armor has been disabled. This makes pvp fresh, interesting and exciting. For the first time ever, we get to balance pvp and make almost every weapon as useful as each other. Damage cheating and godmode has been eradicated. PvP has never been so balanced.
+
+### Build
+TShock Server, `/build` to join.
+
+Build is the dimension where you get to see cool builds or build your own, to build in this dimension, you must have [Crew](#Crew). Anyone that doesn't have crew can build in the sandbox, you can use the warp at spawn or do `/warp sandbox` to tp to sandbox.
+
+### Survival
+TShock Server, `/survival` to join.
+
+Survival is pretty much the generic survival with a few exceptions. Survival resets every week at 3pm - 7pm EST Friday, the world is reset every 12 hours, there are also warps. An account **IS REQUIRED** to play on Survival.
+
+> Q: How can I get unfrozen/unstoned/unwebbed?
+>
+> A: Login to an account, if you don't already have one, follow the steps in [Registering](Accounts.md#Registering).
+
+> Q: What does "The ancient spirits of light and dark are dying." mean?
+>
+> A: It means hardmode is about go away, this is purely a server feature, it is not a feature of Terraria.
+
+> Q: What does "The ancient spirits of light and dark have been vanquished." mean?
+>
+> A: It means hardmode is gone and boss data is wiped, this is purely a server feature, it is not a feature of Terraria.
+
+> Q: How does hardmode die?
+>
+> A: When the mob cap is reached(if you have [Crew](#Crew) you can do `/worlddata mode` to see the cap and the current kill count).
+
+> Q: What are the warps?
+>
+> A: (As of Oct 20 2019) The warps are Elevator, cavern, lhell, rhell, hell, locean, rocean, dungeon, jungle. You can see a list of warps by doing `/warp list`. locean and rocean are both ocean warps, locean is for the left ocean and rocean is for the right ocean. lhell, hell, rhell are all hell warps, lhell is for left hell, hell is a point in hell close to the center of the world, rhell is for right hell.
+
+> Q: Where can I build a house?
+>
+> A: You can build a house on the spawn platform(the entirety of the asphalt runway), if you build it anywhere else, it will get removed by world regen.
```

Comment:
```suggestion
> A: You can build a house on the spawn platform (the entirety of the asphalt runway), if you build it anywhere else, it will get removed by world regen.
```